model/configuration/environmentVariables.config

```diff
@@ -1,7 +1,7 @@
 1 TEMP			298.15
 2 PRESS			1013.25
 3 RH			NOTUSED
-5 H2O			3.91e+17
+4 H2O			3.91e+17
```

Comment:
@spco Is this okay? Could it have caused issues in previous model runs?

Reply:
Good spot! But no, this number is unused - it's read at https://github.com/AtChem/AtChem2/blob/master/src/inputFunctions.f90#L827, and is an unused dummy variable, so we're safe :)